generatesitemap.py

```diff
@@ -298,7 +301,8 @@ def writeXmlSitemap(files, baseUrl, dropExtension=False) :
     sitemapFormat = sys.argv[5]
     additionalExt = set(sys.argv[6].lower().replace(",", " ").replace(".", " ").split())
     dropExtension = sys.argv[7]=="true"
-
+    date_only = sys.argv[8]	
```

Comment:
There's currently a merge conflict due to some refactoring I was in the middle of, and just merged. However, this comment here is still applicable. Specifically, take a look at how the other boolean inputs are converted here from the strings that the Actions framework passes via command line params to actual boolean values, with a string comparison here to the string "true". This enables cleaner logic elsewhere. See my comment somewhere above where this is used.